README.md

```diff
@@ -50,8 +60,19 @@ In this configuration, `seed-isort-config` will adjust the `known_third_party`
 section of the `isort` configuration before `isort` runs!
 
 Note that `seed-isort-config` doesn't act like a normal pre-commit linter so
-exclusion must be configured through `args: [--exclude=...]` instead.
-`--exclude` takes a python regular expression.
+file exclusion must be configured through `args: [--exclude=...]` instead.
+The argument takes a python regular expression.
+
+```yaml
+- repo: https://github.com/asottile/seed-isort-config
+  rev: v1.2.0
+  hooks:
+  - id: seed-isort-config
+    args: [
+      --application-directories, 'src:.',
+      --exclude, 'tests/.*\.py'
+    ]
+```
```

Comment:
in an effort to reduce me forgetting to update the version again :laughing: like #14, let's limit the yaml examples to just include the `args` setting -- I think `exclude` is properly covered though maybe an explicit example After "The argument takes a python regular expression." like `For example: `args: [--exclude=tests/.*\.py]`

Reply:
That makes sense to me. I'll have that fixed right away.

Reply:
Hrm. There's a bit of a gotcha here I just discovered.

```yaml
# the following two are equivalent
args: [--exclude, 'tests/.*\.py']
args: [--exclude=tests/.*\.py]
# this one is not
args: [--exclude='tests/.*\.py']
```

Would you like me to document that?

Reply:
I don't think it's worth documenting quirks of yaml works in this README.  That has nothing to do with this project and is merely because "lol yaml"